packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -5008,7 +5008,11 @@ describe('scope hoisting', function () {
         ],
       },
       {assets: ['dep.js']},
-      {assets: ['async-has-dep.js', 'dep.js', 'get-dep.js']},
+      {
+        assets: process.env.PARCEL_TEST_EXPERIMENTAL_BUNDLER
```

Comment:
We now share assets between workers, so this is an improvement over the prior bundler.

Reply:
But didn't that cause the general problem of `importScripts` being called in non-worker contexts (or vice versa) and also `env.context` being "wrong" in transformers? Didn't you say you disabled this sharing for your apps?